issue_04/script.js

```diff
@@ -0,0 +1,28 @@
+const contentsArray = [{
```

Comment:
IMO
You can tell from the value that it is an array, so the variable name might as well be "contents".

Reply:
It's true that the value shows that it is an array, so we should review the array name. 
Thank you very much.
https://github.com/yoshikawa-akr/js-lessons/pull/4/commits/5febcd6735a7b1bad6987907b8ec43f34b721951